modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItCommonApiTest.java

```diff
@@ -95,4 +108,61 @@ public void testSessionExpiration() throws Exception {
         // second session could start new query
         ses2.execute(null, "SELECT 2 + 2").close();
     }
+
+    /** Check timestamp type operations correctness using sql and kv api. */
+    @Test
+    public void checkTimestampOperations() {
+        String kvTblName = "tbl_all_columns_sql";
+        String schemaName = "PUBLIC";
+        String keyCol = "key";
+        int maxTimePrecision = TemporalColumnType.MAX_TIME_PRECISION;
+
+        Ignite node = CLUSTER_NODES.get(0);
+
+        // TODO: https://issues.apache.org/jira/browse/IGNITE-19162 Trim all less than millisecond information from timestamp
```

Comment:
could we mention IGNITE-15622 instead of IGNITE-19162 and close IGNITE-19162 as duplicate?